src/test/java/org/exmple/utility/UtilityMethodsTest.java

```diff
@@ -96,4 +97,19 @@ public void countEven_shouldThrowNullPointerException() {
 
         assertTrue(actualMessage.contains(expectedMessage));
     }
+    @Test
```

Comment:
Please, add an additional line before the method to visually separate the methods